react/week2/exersice-react/src/components-todo/TodoList.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react'
+import TodoItems from './TodoItems'
+import TodoListItem from './TodoListItem'
+
+function TodoList() {
+    const originalTodos = TodoItems();
+    let [updatedTodos, setTodos] = useState(originalTodos);
```

Comment:
You won't need this function anymore and please care the name conventions. 
```suggestion
    const [todos, setTodos] = useState(todoList);
```